lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -465,6 +465,11 @@ function dosomething_user_new_user($form, &$form_state) {
   global $user;
   $account = $user;
 
+  // Adjust language based on location
+  $user_country_code = dosomething_settings_get_geo_country_code();
```

Comment:
wrap this in `if (module_exists('dosomething_global')` so it doesn't fail on prod